v1.0.0

iNoahNothing opened the pull request:
- Remove install scripts
- Update CRDs to getambassador.io/v2
- Retest everything so it still works
- Update configs that rotted

(Should have done this in more than one commit oops)

Signed-off-by: Noah Krause <krausenoah@gmail.com>

iNoahNothing committed:
Update pro-ref-arch for AES

- Remove install scripts
- Update CRDs to getambassador.io/v2
- Retest everything so it still works
- Update configs that rotted

(Should have done this in more than one commit oops)

Signed-off-by: Noah Krause <krausenoah@gmail.com>

iNoahNothing commented:
TODO: Update README.md to state renewed purpose

kflynn commented on api-auth-with-github/README.md:
```diff
@@ -2,21 +2,21 @@
 
-1. Install Ambassador Pro: `make apply-ambassador`.
-2. Create an OAuth application in GitHub.
+1. [Install Ambassador Pro](https://www.getambassador.io/user-guide/install).
```
Ambassador Pro?

iNoahNothing replied:
Nope! Thanks!

kflynn commented on ratelimit/README.md:
```diff
@@ -2,73 +2,50 @@
+## Prerequesites
 
-Suppose we want to limit a certain subset of users to only 10 requests per minute for any requests through Ambassador. We can configure a global rate limit that can rate limit based off a header that identifies this subset of users. Users with the header `x-limited-user: true` will be limited to 10 requests per minute. 
+1. [Install Ambassador Edge Stack](https://www.getambassador.io/early-access/user-guide/getting-started/)
```
Should this reference the EA URL or the real URL?

iNoahNothing replied:
Probably real. Thanks!

kflynn reviewed (changes requested):
Two questions, I think it looks OK otherwise? I'm trusting Noah to have actually followed the docs in here. 🙂

iNoahNothing committed:
Minor README updates

Signed-off-by: Noah Krause <krausenoah@gmail.com>

khussey commented on env.sh.example:
```diff
@@ -1,24 +1,9 @@
+CLIENT_SECRET=49cd212341c82f797502720bf9ec9d0bb11676d6
 
+## Configuration settings for monitoring configuration
+AMBASSADOR_URL=aes.nkrause.k736.net
```
@nbkrause is it OK that your name is here?

khussey commented on grpc-web/ambassador-module.yaml:
```diff
@@ -0,0 +1,8 @@
+  name: ambassador
+spec:
+  config:
+    enable_grpc_web: True
```
Is the special character after True here OK?

khussey commented on ratelimit/quote-backend-mapping.yaml:
```diff
@@ -0,0 +1,13 @@
+  labels:
+    ambassador:
+      - remote_address_label_group:
+        - remote_address
```
Is the special character at the end here OK?

khussey commented on ratelimit/rl-per-user.yaml:
```diff
@@ -5,6 +5,6 @@ metadata:
-   - pattern: [{x_limited_user: "false"}, {remote_address: "*"}]
+   - pattern: [{remote_address: "*"}]
      rate: 20
      unit: minute
```
Another special character here?

khussey reviewed (commented):
@nbkrause I made some comments, not sure if they are real concerns. Otherwise I am +1.

iNoahNothing committed:
Some more updates

- Remove tracing since it does not work at the moment (example is
broken)
- Update README

Signed-off-by: Noah Krause <krausenoah@gmail.com>

iNoahNothing committed:
minor fixes

Signed-off-by: Noah Krause <krausenoah@gmail.com>

kflynn reviewed (approved):
Looks OK I think.

khussey reviewed (approved):
+1

LukeShu commented:
Why the removal of `tracing/`?

LukeShu commented:
Also: Nice job! This is *waaaay* more thorough  than I would have been.
